src/Oracle/Oracle/config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "PluginConfiguration": {
+    "Nodes": [],
+    "Wallet": "",
+    "MaxTaskTimeout": 432000000,
+    "HttpTimeout": 5000,
```

Comment:
we should define `HttpTimeout` in the smart contract

Reply:
Erik suggests putting it in this configuration file here.